src/classes/BDI_DataImportService.cls

```diff
@@ -691,9 +691,10 @@ global with sharing class BDI_DataImportService {
         String paymentStatus = dataImport.Payment_Status__c;
 
         return String.isBlank(paymentStatus) ||
-            dataImport.Payment_Status__c.equalsIgnoreCase(
+            (dataImport.Payment_Status__c.equalsIgnoreCase(
```

Comment:
This change will be reversed once this #6108 merges to master. It's here so DIs with ACH transactions get processed.